app/modules/utils/logger.py

```
@@ -102,6 +103,42 @@ def filter_sensitive_data(text: str) -> str:
+    "password",
+    "passwd",
+    "secret",
+    "access_token",
+    "refresh_token",
+    "id_token",
+    "api_key",
+    "apikey",
+)
+
+
+def filter_sensitive_value(value, key: str | None = None):
```

Ruff (0.16.0)

[warning] Too many return statements (8 > 6)

(PLR0911)